docs/content/docs/ops/metrics.md

```diff
@@ -1292,6 +1292,11 @@ Note that for failed checkpoints, metrics are updated on a best efforts basis an
       <td>The checkpointed size of the last checkpoint (in bytes), this metric could be different from lastCheckpointFullSize if incremental checkpoint or changelog is enabled.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
```

Comment:
"rowspan" should also be modified so that the format of the table could be showed correctly.
You could build the doc locally to check it.

Reply:
Thank you for pointing out. I have corrected it